automations/js/package.json

```diff
@@ -5,9 +5,9 @@
   "version": "0.0.0",
   "dependencies": {
     "@octokit/rest": "19.0.7",
+    "axios": "^0.27.0",
```

Comment:
Can't automations use the last version of Axios?

Reply:
I stuck to the version used in the frontend to prevent conflicts in shamefully hoisted dependencies.